CHANGELOG.md

```diff
@@ -35,8 +35,8 @@ Full documentation for MIGraphX is available at [MIGraphX Documentation](https:/
 - Fix compile warnings for shadowing variable names
 - Add missing specialization for the `nullptr` for the hash function
 ### Changed
-- Bumped version of half library to 5.6.0
-- Bumped CI to support rocm 5.6
+- Bumped version of half library to 5.7.0
+- Bumped CI to support rocm 5.7
```

Comment:
This shouldn't be changed i think

Reply:
Do we save changelogs then for version bumps of our libs and not ROCm release/

Reply:
This `changelog.md` is for rocm release 5.7. it is already released. I don't think it should be changed now.

Reply:
Ill remove this then

Reply:
Done